test/ProgressOnderwijsUtils.Tests/HtmlTests.cs

```diff
@@ -70,7 +70,7 @@ public void AppendingThreeThingsWorks()
         public void PerfIterativeAppendingDoesNotCreateLotsOfArrays()
         {
             var html = "1".AsFragment().Append("2").Append("3").Append("4");
-            var kids = (HtmlFragment[])html.Implementation;
+            var kids = (HtmlFragment[])html.Implementation!;

```

Comment:
So casts are implicitly nullable even though the cast-to-type is not?  Oh well.

Reply:
I don't think so? '!' is necessary because `Implementation` is nullable.

Reply:
But that doesn't actually make sense; a case is a type-assertion, so now we need double-type-assertions.  Apparently that's how it works, but it's redundant conceptually.

Reply:
(it may be due to migrations issues for existing codebases)